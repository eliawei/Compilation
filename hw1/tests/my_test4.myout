1 STRING Hex value jFj
2 ID blah
Error "
Error \
2 NUM 0
Error \
2 NUM 0
2 ID string
Error \
2 ID q
Error "
