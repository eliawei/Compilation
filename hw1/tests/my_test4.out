1 STRING Hex value jFj
2 ID blah
Error undefined escape sequence q
